environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - enum34
   - ffmpeg
   - flake8>=2.1.0
-  - gcc
+  - git
   - jupyter
```

Comment:
Please don't delete gcc from the environment. It is needed to stay independent of gcc from the operating system on Linux and Mac. As there is no gcc package for Windows, there should be a separate set of dependencies. Consider to add a file environment_windows.yml?